src/page-response-impact/ReducingR0ImpactMetrics.tsx

```diff
@@ -1,41 +1,51 @@
 import React from "react";
-import styled from "styled-components";
 
-import downArrowIcon from "./icons/ic_down_arrow.svg";
 import hospitalIcon from "./icons/ic_hospital.svg";
 import staffIcon from "./icons/ic_staff.svg";
 import ImpactMetricCard from "./ImpactMetricCard";
 import ImpactMetricsContainer from "./ImpactMetricsContainer";
+import {
+  getSubtitle,
+  ImpactTitleProps,
+  ImpactTitleSpan,
+} from "./utils/ResponseImpactCardStateUtils";
 
-const Icon = styled.img`
-  display: inline;
-  margin-right: 15px;
-`;
-
-const PercentValue = ({ value }: { [value: string]: string }) => {
-  // Note: This assumes the icon should always be a down arrow, should confirm
+const HospitalBedsTitle = ({ title, value }: ImpactTitleProps) => {
+  const valueSign = Math.sign(value);
+  const subtitle = getSubtitle(valueSign);
   return (
-    <div>
-      <Icon src={downArrowIcon} alt="down arrow icon" />
-      <span>{value}</span>
-    </div>
+    <>
+      <span>{title}</span>{" "}
+      <ImpactTitleSpan color={subtitle.color}>{subtitle.text}</ImpactTitleSpan>
+    </>
   );
 };
 
 const ReducingR0ImpactMetrics: React.FC = () => {
   // TODO: Replace with actual data
+  const staffAbleToWork = 200;
+  const percentageBedsUsed = 50;
+  const percentageOfStaff = 12;
+  const percentageOfIncarcerated = 12;
+  const staffAbilityText = Math.sign(staffAbleToWork) === 1 ? "able" : "unable";
+
   return (
     <ImpactMetricsContainer>
       <ImpactMetricCard
-        title="Hospital beds used"
-        value={<PercentValue value={"50%"} />}
-        subtitle="12% of staff"
+        title={
+          <HospitalBedsTitle
+            title="Hospital beds used at peak"
+            value={percentageBedsUsed}
+          />
+        }
+        value={`${Math.abs(percentageBedsUsed)}%`}
+        subtitle={`${Math.abs(percentageOfStaff)}% of staff`}
```

Comment:
We don't need this subtitle anymore

Reply:
Actually, I'll remove it in my PR. Continue :)

Reply:
Unless I'm looking at the wrong version of the mocks. Looks to me like there is no longer a subtitle, right?

Reply:
I think you're right if it's the right mocks! @sychang when you have a moment could you confirm that this is the right mock for the bottom cards? It looks like the arrow next to the percentage was removed and the percentages of staff/incarcerated text are removed too. 

![image](https://user-images.githubusercontent.com/5402804/81621565-965f4600-93a3-11ea-8282-a21c035e65f9.png)